integration_annotation_manager.py

```diff
@@ -1,7 +1,11 @@
 #%%
 import json
+import os
 from tkinter import filedialog
 
+import cv2
+import cytoBoom as CB
```

Comment:
please dont use cytoBoom: 
1. it is a private module so this can not be used outside 
2. cytoboom depends on tomni so this will give circular imports